lib/src/views/Clock/Clock.tsx

```diff
@@ -95,7 +96,7 @@ export const useStyles = makeStyles(
   muiComponentConfig
 );
 
-export const Clock: React.FC<ClockProps> = withDefaultProps(muiComponentConfig, (props) => {
+export function Clock<TDate>(props: ClockProps<TDate>) {
```

Comment:
Important question. What would it take to use makeStyles here?

Reply:
I didn't get the question. What the problem of using `makeStyles` could be?

Reply:
We will need to use the HOC API for integrating the components back into the lab. So we might as well do it now.

Reply:
No we cannot use HOCs with pickers sources.

Reply:
In this case, we need to reconsider the approach, what about we expose a native Date object? It's the approach used by https://github.com/adobe/react-spectrum/blob/c700898916bbd076bcc63e49d77c16d80623a8e7/packages/%40react-spectrum/calendar/src/Calendar.tsx (even if they are using date-fns), it seems simpler :)

Reply:
Also, I wonder, isn't the Autocomplete using generic and withStyles at the same time?

Reply:
Generics lives only on the typescript side – autocomplete written in js. 

it is possible to use withStyles. Simply by adding `withStyles() as typeof CalendarView` where `CalendarView` is generic. BUT why? I would be happy to make lab infrastructure to work without `withStylea`

Reply:
> BUT why?

A couple of reasons that come to mind:

1. @mnajdova Is working on revamping how we write the components (at the style and unstyled level). I think that normalizing the approach will make her job easier. 
2. It bundles a couple of behavior that forces us to implement, which is great. `classes` API, theme default prop behavior.
3. It's required for the API classes markdown generation to work correctly.
4. It's required for the tests getClasses utility to work correctly.

> I would be happy to make lab infrastructure to work without withStylea

I think that we should leave this to 1. and focus on the migration for now. But if you have insight, please raise them to Marija :). Also, I think that we should start the migration from where we have the most constraints.